po/nl.po

```diff
@@ -189,7 +189,7 @@ msgstr "Kan niet verbinden met ratbagd"
 #. Translators: "it" here is ratbagd.
 #: piper/window.py:57
 msgid "Please make sure it is running"
```

Comment:
that's how I got to the german one. sorry, the software engineer in me refused to run a whole update for a single line change :) Anyway, fixed with your follow-up commits, thanks.